ocfweb/tv/templatetags/tv_formatting.py

```diff
@@ -0,0 +1,14 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def tv_lab_hours(hour):
+    if hour:
+        if hour.open.minute != 0 or hour.close.minute != 0:
+            fmt = '{:%-I:%M%P}-{:%-I%M%P}'
+        else:
+            fmt = '{:%-I%P}-{:%-I%P}'
+
+        return fmt.format(hour.open, hour.close)
```

Comment:
I modeled this after that, that one joins everything into one string but that doesn't display very cleanly on the TV, as in it didn't break nicely, so I rewrote it to just format the time, while the loop in the template automatically breaks multiple hours so that everything is displayed cleanly